Bump version to v0.3.0 (#661)
open-mmlab/mmengine: docs/en/notes/changelog.md

# Changelog of v0.x

## v0.3.0 (11/02/2022)

### New Features & Enhancements

> - Support running with NPU backend. by @wangjiangben-hw in https://github.com/open-mmlab/mmengine/pull/572

Reviewer: ```suggestion
- Support running on Ascend chip by @wangjiangben-hw in https://github.com/open-mmlab/mmengine/pull/572
```

- Support torch `ZeroRedundancyOptimizer` by @nijkah in https://github.com/open-mmlab/mmengine/pull/551
- Add non-blocking feature to `BaseDataPreprocessor` by @shenmishajing in https://github.com/open-mmlab/mmengine/pull/618

> - Add documents for `clip_grad` , and support clip grad by value. by @HAOCHENYE in https://github.com/open-mmlab/mmengine/pull/513

Reviewer: ```suggestion
- Add documents for `clip_grad`, and support clip grad by value. by @HAOCHENYE in https://github.com/open-mmlab/mmengine/pull/513
```

# Changelog of v0.x

## v0.3.0 (11/02/2022)

### New Features & Enhancements

- Support running with NPU backend. by @wangjiangben-hw in https://github.com/open-mmlab/mmengine/pull/572
- Support torch `ZeroRedundancyOptimizer` by @nijkah in https://github.com/open-mmlab/mmengine/pull/551
- Add non-blocking feature to `BaseDataPreprocessor` by @shenmishajing in https://github.com/open-mmlab/mmengine/pull/618
- Add documents for `clip_grad` , and support clip grad by value. by @HAOCHENYE in https://github.com/open-mmlab/mmengine/pull/513
- Add ROCm info when collecting env by @zhouzaida in https://github.com/open-mmlab/mmengine/pull/633
- Add a function to mark the deprecated function. by @HAOCHENYE in https://github.com/open-mmlab/mmengine/pull/609
- Call `register_all_modules`  in `Registry.get()` by @HAOCHENYE in https://github.com/open-mmlab/mmengine/pull/541
- Deprecate `_save_to_state_dict` implemented in mmengine by @HAOCHENYE in https://github.com/open-mmlab/mmengine/pull/610
- Add `ignore_keys` in ConcatDataset by @BIGWangYuDong in https://github.com/open-mmlab/mmengine/pull/556

> - Add issue or pr assignees by @zhouzaida in https://github.com/open-mmlab/mmengine/pull/608

Reviewer 2: unnecessary, delete this

> - Add `CODEOWNERS` file for auto assigning reviewer by @HAOCHENYE in https://github.com/open-mmlab/mmengine/pull/605

Reviewer 2: unnecessary

### Docs

> - Fix typo in `docs/zh_cn/tutorials/model.md` by @C1rN09 in https://github.com/open-mmlab/mmengine/pull/598

Reviewer 2: Put typos at the end of the section.

- Fix typo in `docs/zh_cn/tutorials/config.md` by @Xiangxu-0103 in https://github.com/open-mmlab/mmengine/pull/596
- Fix cannot show `changelog.md` in chinese documents. by @HAOCHENYE in https://github.com/open-mmlab/mmengine/pull/606
- Fix typo in `docs/zh_cn/advanced_tutorials/visualization.md` by @MambaWong in https://github.com/open-mmlab/mmengine/pull/616
- Fix Chinese docs whitespaces by @C1rN09 in https://github.com/open-mmlab/mmengine/pull/521
- Translate installation and 15_min by @xin-li-67 in https://github.com/open-mmlab/mmengine/pull/629
- Refine chinese doc by @Tau-J in https://github.com/open-mmlab/mmengine/pull/516

> - Add mmyolo link in README by @Xiangxu-0103 in https://github.com/open-mmlab/mmengine/pull/634

Reviewer 2: mmyolo -> MMYOLO
